solr/licenses/slf4j-api-1.7.25.jar.sha1

```diff
@@ -0,0 +1 @@
+da76ca59f6a57ee3102f8f9bd9cee742973efa8a
```

Comment:
It's strange that we are getting 1.7.25 (here) and also 1.7.30 resolved...

Reply:
Good catch - most likely transitive dependency resolution across all the modules. Its easy to force to 1.7.30 in versions.props. I didn't bump it but probably should.